src/material/schematics/ng-generate/navigation/schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/schema",
-  "id": "SchematicsMaterialNav",
-  "title": "Material Nav Options Schema",
+  "id": "SchematicsMaterialNavigation",
+  "title": "Navigation schematic options schema",
```

Comment:
Maybe this should still say "Material"? Otherwise it's not obvious where it's coming from.

Reply:
I don't think that any tool uses the `schema.json` title. I will update it, but it shouldn't have any effect since the schematic name/description is part of the schematic collection.

Reply:
Updated.